components/blocks/cards/LogInCard/LogInCard.test.ts

```diff
@@ -60,6 +95,43 @@ describe('<LogInCard />', () => {
       expect(emailInput.value).toBe('')
       expect(passwordInput.value).toBe('')
     })
+
+    it('calls the api', async () => {
+      const { getByTestId } = stubbedRender(LogInCard)
+
+      const emailInput: HTMLInputElement = getByTestId('email-input')
+      const passwordInput: HTMLInputElement = getByTestId('password-input')
+
+      await fireEvent.type(emailInput, emailAddress)
+      await fireEvent.type(passwordInput, password)
+
+      await fireEvent.click(getByTestId('login-button'))
+
+      const apiCalls = fetchMock.mock.calls as fetchMockCall[]
+      expect(apiCalls.length).toBe(2)
+
+      const loginApiCall = apiCalls[0]
+      expect(loginApiCall?.[0]).toBe(
+        `${process.env.BACKEND_BASE_URL}/api/Users/login`,
+      )
+      expect(loginApiCall?.[1].method).toBe('POST')
+      expect(loginApiCall?.[1].body).toEqual(
+        JSON.stringify({
+          email: emailAddress,
+          password,
+        }),
+      )
+
+      const meApiCall = apiCalls[1]
+      expect(meApiCall?.[0]).toBe(
+        `${process.env.BACKEND_BASE_URL}/api/Users/me`,
+      )
+      expect(meApiCall?.[1].method).toBe('GET')
+      expect(meApiCall?.[1].headers).toBeDefined()
+      expect(Object.keys({ ...meApiCall?.[1].headers })).toContain(
```

Comment:
Can we not call `expect(Object.keys(meApiCall?.[1].headers)).toContain(...` here?